libs/designsystem/src/lib/components/list/directives/infinite-scroll.directive.spec.ts

```diff
@@ -69,6 +72,19 @@ describe('InfiniteScrollDirective', () => {
       expect(directive.scrollEnd.emit).toHaveBeenCalledTimes(1);
     }));
 
+    it('should emit event if initalized as disabled and then enabled while more than 80% of the element has been scrolled', fakeAsync(() => {
```

Comment:
The wording on this is a bit clumsy - I will change it if I come up with something better. Suggestions would be appreciated tho 😀